sklearn/datasets/openml.py

```diff
@@ -563,4 +569,7 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         categories=nominal_attributes,
         url="https://www.openml.org/d/{}".format(data_id))
 
+    if return_X_y:
```

Comment:
Maybe put it above ``bunch = Bunch(...)``

Reply:
Yes, please do.